lucida/commandcenter/controllers/Config.py

```diff
@@ -12,41 +12,163 @@
 # models are already saved in `../models`.
 TRAIN_OR_LOAD = 'train' # either 'train' or 'load'
 
+
+
+
+
+#Contains serviceName and data to pass. Needed for batch (and thereby parallel) processing.
+class serviceRequestData(object):
+
+	def __init__(self,nameOfService,argData):
+		self.argumentData = argData
+		self.serviceName = nameOfService
+	
+
+class workFlow(object):
+		def __init__(self):
+			self.currentState = 0; # What state on the state graph
+			self.isEnd = False;
+			self.batchedData = []
+	
+	
+	
+	
+
+class firstWorkflow(workFlow):
+
+	
+	def processCurrentState(self,inputModifierText,inputModifierImage):
+		print "Executing state logic";
+		
+		if(self.currentState==0):
+			print "State 0";
+			self.currentState = 1;
+			self.batchedData = [serviceRequestData("QA",[unicode("How old is Johann")]),serviceRequestData("QA",[unicode("What color is a pug?")])];
```

Comment:
That is the demo workflow. That workflow doesn't take any input (Well it takes input, just doesn't use it) and is just to demonstrate how batching and a complex workflow works.